src/QuantumToolbox.jl

```diff
@@ -3,17 +3,19 @@ module QuantumToolbox
 # Re-export:
 #   1. StaticArraysCore.SVector for the type of dims
 #   2. basic functions in LinearAlgebra and SparseArrays
+#   3. some functions in SciMLOperators
 import Reexport: @reexport
 @reexport import StaticArraysCore: SVector
 @reexport using LinearAlgebra
 @reexport using SparseArrays
+@reexport import SciMLOperators: cache_operator, iscached, isconstant
```

Comment:
But as you can see in the test I added.
even with that simple case, we need to cache operator manually

Reply:
Ok just because we are using the Lazy tensor operator, which we should avoid in general. 

Let say that we reexport this, but we should avoid this as much as we can, because these functions then become as part of QuantumToolbox.jl instead of the parent package, which might lead to confusion.

Reply:
Yeah
the reason I implement this method is because. When people saw the un-cached error message from SciMLOperators. The message suggests people to do `L = cache_operator(L, u)`